jupyterhub/apihandlers/users.py

```diff
@@ -193,6 +197,52 @@ def delete(self, name):
         status = 202 if user.stop_pending else 204
         self.set_status(status)
 
+
+class UserCreateMultiServerAPIHandler(APIHandler):
+    """Create multi single-user server
+    
+    This handler should be used when c.JupyterHub.allow_multiple_servers = True
+    """
+    @gen.coroutine
+    @admin_or_self
+    def post(self, name):
+        user = self.find_user(name)
```

Comment:
I guess is not really possible.

I tried to mimic this: https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/apihandlers/users.py#L163

With the only thing that doesn't check if a server is already running, but for sure it needs more work this part.